### PR TITLE
contactus page is added

### PR DESCRIPTION
**Pull request by sharondev11**
-- added contactus

### CODE REVIEW

File: contactus.html
Comment (sharondev11) [inline contactus.html:1]: where is alt

**Review by Renil711 — COMMENTED** [review]
hgmn

**Review by Renil711 — CHANGES_REQUESTED** [review]
ftgyhu

**Review by sharondev11 — COMMENTED** [review]
Hi, 
please check my review comments

**Review by Renil711 — APPROVED** [review]

**Pull request merged**